pkg/provider/azure_storageaccount.go

```diff
@@ -212,7 +216,17 @@ func (az *Cloud) EnsureStorageAccount(ctx context.Context, accountOptions *Accou
 			}
 
 			if len(accounts) > 0 {
-				accountName = accounts[0].Name
+				index := 0
+				if accountOptions.PickRandomMatchingAccount {
+					// randomly pick one matching account
+					n, err := rand.Int(rand.Reader, big.NewInt(int64(len(accounts))))
```

Comment:
Suggest that use `
```go
rand.Seed(time.Now().UnixNano())
index := rand.Intn(len(accounts))
```
 to get the index.

Reply:
that's my first version, while `rand` package is already deprecated by golint check, so I switched to `crypto/rand`